src/css/jquery-usos.css

```diff
@@ -1445,3 +1472,37 @@ a.ua-close-link:hover { background: rgba(105,105,105,0.45); }
 .ua-icon-firefox:before {
     content: "\e7be";
 }
+
+/* Hide all tooltip arrow faces that are not displayed */
+
+.ua-tooltip.tooltipster-right .tooltipster-arrow-background,
+.ua-tooltip.tooltipster-right .tooltipster-arrow-border {
+    border-left-color: transparent;
+    border-top-color: transparent;
+    border-bottom-color: transparent;
+    //border-right-color: transparent;
```

Comment:
https://stackoverflow.com/questions/12298890/is-it-bad-practice-to-comment-out-single-lines-of-css-with